src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -394,4 +394,10 @@
   <data name="InputFileNotSpecified" xml:space="preserve">
     <value>Please specify a nuspec, project.json, project file to use</value>
   </data>
+  <data name="Error_PackFailed" xml:space="preserve">
+    <value>Error packaging '{0}'.</value>
+  </data>
+  <data name="Error_InvalidTargetFramework" xml:space="preserve">
+    <value>Unsupported targetFramework value on '{0}'</value>
```

Comment:
I've been learning more about project.json and I've noticed that almost all of the error messages are terrible and don't help me understand where the error is and why there is an error.

---
In reply to: [60457715](https://github.com/NuGet/NuGet.Client/pull/514#discussion_r60457715) [](ancestors = 60457715)